src/Processor/TrimProcessor.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the BibTex Parser.
+ *
+ * (c) Florent DESPIERRES <florent@despierres.pro>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace RenanBr\BibTexParser\Processor;
+
+class TrimProcessor
```

Comment:
Add `TagCoverageTrait`, then user can choose the tags they want to trim or not.

Like in #70